Sources/SwerverCore/Action/ActionProtocol.swift

```diff
@@ -0,0 +1,3 @@
+public protocol Action {
```

Comment:
I feel like this could use a different name than "Action"

Reply:
will try to come up with a couple potential names for this protocol